examples/README.md

```diff
@@ -1,12 +1,13 @@
 ## Examples
 
 This directory contains small example programs that you can copy to get started quickly. Try these examples out for
-yourself by cloning (or downloading) this repository, running `npm install` in this directory, and running one of the
-scripts (e.g. `node basic-web-client.js`). We recommend using the latest LTS version node.
+yourself by cloning (or downloading) this repository, running `npm install` in this directory. Then run one of the
```

Comment:
"Try these examples out for yourself by cloning (or downloading) this repository and running `npm install` in this directory"